app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -8,7 +8,7 @@ def ems_inv_to_hashes(inventory)
         get_builds(inventory)
         get_build_pods(inventory)
         get_templates(inventory)
-        get_openshift_images(inventory)
+#       get_openshift_images(inventory)
```

Comment:
If we end up making this configurable we can do something similar to what aws does [here](https://github.com/ManageIQ/manageiq-providers-amazon/blob/master/app/models/manageiq/providers/amazon/cloud_manager/refresh_parser.rb#L26)